KDCircularProgress/KDCircularProgress.swift

```diff
@@ -144,7 +144,7 @@ public class KDCircularProgress: UIView, CAAnimationDelegate {
     
     @IBInspectable public var progressThickness: CGFloat = 0.4 {//Between 0 and 1
         didSet {
-            progressThickness = progressThickness.clamped(toMinimum: 0, maximum: 1)
+            progressThickness = progressThickness.clamped(toMinimum: 0, maximum: 10)
```

Comment:
Why 10 here? Won't we need any values greater than 10. Can you also modify the comment above so that it says `Between 0 and 10`.

Also, don't we also want to be consistent and increase the range to `0...10` for `trackThickness` as well?

Reply:
I didn't have a specific reason to use 10. I just knew that in my implementation I often was bumping this up to give a higher maximum than 1. 10 is just a general round number which gives a good amount of buffer room.

Also good call on the comment -- I will update it.